.codecov.yml

```diff
@@ -1,6 +1,3 @@
-# Don't post a comment on pull requests.
```

Comment:
Intentional deletion?

Reply:
Sorry, I was testing changes.  I did not think the PR was ready for review yet.